indico/web/client/js/react/components/principals/PrincipalListField.jsx

```diff
@@ -164,6 +166,7 @@ const PrincipalListField = props => {
               )}
             </>
           )}
+          {renderPluginComponents('setManagersButton', {fieldId, eventId})}
```

Comment:
- Most of those signal names use kebab-case capitalization (even though it's pretty inconsistent atm :/)
- The name is way too generic, something like `principal-list-field-add-buttons` would be more suitable